packages/studio/tests/store/StudioActions/activeComponentActions.test.ts

```diff
@@ -49,6 +52,29 @@ describe("updateComponentTree", () => {
   });
 });
 
+describe("updateActiveComponentProps", () => {
```

Comment:
would you consider these describe blocks redundant?  I kind of like them so we know the purpose and what exact function it is intended to test-- but I would understand why they would be seen as clutter

Reply:
I like the describe blocks, I'd vote to keep them.